addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -103,8 +103,11 @@ const Wysiwyg = Widget.extend({
                 }
             },
             isHintBlacklisted: node => {
-                return node.hasAttribute &&
-                    (node.hasAttribute('data-target') || node.hasAttribute('data-oe-model'));
+                return node.hasAttribute && (
+                    node.hasAttribute('data-target') ||
+                    node.hasAttribute('data-oe-model') ||
+                    node.classList.contains('nav-item')
```

Comment:
Indeed. I introduced a parameter that makes it possible to define additional blacklisted nodes so that the `.nav-item` rule is only applied for the editor used in website.